<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="categoryComboBox">
        <item>
         <property name="text">
          <string>Distance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Weight</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="toggleButton">
        <property name="text">
         <string>Swap</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="type1ComboBox"/>
      </item>
      <item>
       <widget class="QComboBox" name="type1PrefixComboBox">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>M</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>k</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>d</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>c</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>m</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>μ</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="type2ComboBox"/>
      </item>
      <item>
       <widget class="QComboBox" name="type2PrefixComboBox">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>M</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>k</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>d</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>c</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>μ</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDoubleSpinBox" name="minSpinBox">
        <property name="minimum">
         <double>-1000000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000000.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="currSpinBox">
        <property name="minimum">
         <double>-1000000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000000.000000000000000</double>
        </property>
        <property name="value">
         <double>50.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="maxSpinBox">
        <property name="minimum">
         <double>-1000000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000000.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="valueSlider">
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultsLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
